src/common/gui/editor/Editor.ts

```diff
@@ -266,15 +271,26 @@ export class Editor implements ImageHandler, Component {
 		this.styles.i = this.squire.hasFormat("i")
 	}
 
-	makeLink() {
+	/**
+	 * Prompts the user to input a URL and creates a link at the current cursor position in the editor.
+	 *
+	 * @param [areRelativeLinksAllowed=false] - Whether relative links are allowed.
+	 */
+	makeLink(areRelativeLinksAllowed: boolean = false) {
```

Comment:
is there a reason to pass `areRelativeLinksAllowed` in or can we just read it from inside?
I also think that jsdoc for that param doesn't add much and I would just remove it.